File: jscomp/test/key_word_property.js
'use strict';

var Vscode = require("vscode");
var SomeEs6Module = require("some-es6-module");

var $$default = SomeEs6Module.default;

var $$window = Vscode.window;

function mk($$window, $$default) do
  return do
          window: $$window,
          default: $$default
        end;
end

function mk2($$window, $$default) do
  return --[ :: ]--[
          do
            window: $$window,
            default: $$default
          end,
          --[ [] ]--0
        ];
end

function des(v) do
  return do
          window: v.window,
          default: v.default
        end;
end

var test = do
  case: 3,
  window: 3
end;

function u(param) do
  return $$window.switch();
end

var $$case = 3;

exports.$$default = $$default;
exports.default = $$default;
exports.__esModule = true;
exports.$$window = $$window;
exports.mk = mk;
exports.mk2 = mk2;
exports.des = des;
exports.$$case = $$case;
exports.test = test;
exports.u = u;
--[ default Not a pure module ]--

File: jscomp/test/key_word_property.lua
__console = {log = print};

Vscode = require "vs";
SomeEs6Module = require "some-es6-mo";

__default = SomeEs6Module.__default;

__window = Vscode.__window;

function mk(__window, __default) do
  return {
          window = __window,
          default = __default
        };
end end

function mk2(__window, __default) do
  return --[[ :: ]]{
          {
            window = __window,
            default = __default
          },
          --[[ [] ]]0
        };
end end

function des(v) do
  return {
          window = v.__window,
          default = v.__default
        };
end end

test = {
  case = 3,
  window = 3
};

function u(param) do
  return __window.__switch();
end end

__case = 3;

exports = {};
exports.__default = __default;
exports.default = __default;
exports.__esModule = true;
exports.__window = __window;
exports.mk = mk;
exports.mk2 = mk2;
exports.des = des;
exports.__case = __case;
exports.test = test;
exports.u = u;
return exports;
--[[ default Not a pure module ]]
